wgpu-types/src/lib.rs

```diff
@@ -1414,6 +1414,8 @@ pub struct AdapterInfo {
     pub driver_info: String,
     /// Backend used for device
     pub backend: Backend,
+    /// Supports [`TextureUsages::TRANSIENT`]
+    pub supports_transient: bool,
```

Comment:
```suggestion
    pub native_transient_texture: bool,
```

Really not set on this name, but what I'm trying to avoid is the implication that TRANSIENT is not allowed